### DISCUSSION TITLE
Show Trivy integration in Website

### DISCUSSION THREAD

**Discussion by Lingom-KSR** [aquasecurity/trivy#1196] [Q&A] [answered]
Hi,
I'm planning to integrate trivy with my product for container scanning. Just would like to know if I can add 'Trivy' name in the product website saying that "we have integrated with Trivy".

Please let me know about this.

Thanks!

**Comment by knqyf263** [comment, marked as answer]
Sure!

**Reply by Lingom-KSR** [reply]
Thank you for your prompt response. Hope, it would not require any email going to Trivy for an acknowledgment. If so, please help me with the process.

Thanks Again!

**Reply by itaysk** [reply]
@Lingom-KSR Feel free to mention that you have integrated with Trivy without acknowledgement. 
It would be nice to hear about what is your product and how you use Trivy if you don't mind sharing :)
